src/vs/workbench/common/theme.ts

```diff
@@ -675,11 +669,17 @@ export const DASHBOARD_WIDGET_SUBTEXT = registerColor('dashboardWidget.subText',
 	hc: '#FFFFFF'
 }, nls.localize('dashboardWidgetSubtext', "Color for dashboard widget subtext"));
 
-export const DASHBOARD_PROPERTIES_NAME = registerColor('dashboardWidget.propertiesName', {
+export const PROPERTIES_CONTAINER_PROPERTY_VALUE = registerColor('propertiesContainer.propertyValue', {
```

Comment:
This is changing the color IDs but that should be fine since these were new values introduced anyways so no one should be using them.